website/client/js/services/userServices.js

```diff
@@ -310,6 +310,19 @@ angular.module('habitrpg')
           });
         },
 
+        setCustomDayStart: function (dayStart) {
+          $http({
+            method: "POST",
+            url: 'api/v3/user/set-custom-day-start',
+            data: {
+              dayStart: dayStart
+            }
+          })
+          .then(function (response) {
+            Notification.text('Day start updated. Remember to refresh');
```

Comment:
I just copied from the debug function... Seems to me the concept is the similar. I don't think they need to refresh, but I figured you added the message for a reason.